src/linter.js

```diff
@@ -11,7 +11,7 @@ function lint(files, configuration) {
     var file = fs.readFileSync(fileName, 'utf-8');
     var errors = [];
     try {
-      var parsedFile = parser.parse(file);
+      var parsedFile = parser.parse(file).feature || {};
```

Comment:
to check for the empty-file rule these lines could be altered to something like this

```js
      var parsedFile = parser.parse(file).feature;
      if (!parsedFile) {
        errors.push({
          message: 'Empty feature files are disallowed',
          rule: 'no-empty-file',
          line: '0'
        });
      } else {
        errors = rules.runAllEnabledRules(parsedFile, fileName, configuration);
      }
```

And then remove the check at the bottom:
```js
} else if(error.message.indexOf('(1:0): unexpected end of file') > -1) {
```